R/sfi_plot_laqueur_1.R

```diff
@@ -14,101 +14,140 @@ sfi_plot_laqueur_1 <- function(){
-               aes(x = year, 
-                   y = number_of_grants)) +
-    geom_smooth(method = 'lm',
-                size = 0.5, 
-                alpha = 0.6,
+  g4 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = number_of_grants)) +
+    geom_smooth(method = 'loess',
+                alpha = 0.7,
                 color = 'black',
-                linetype = 2) +    
-    geom_point(size = 3, 
-               alpha = 0.6) +
+                linetype = 0) + 
+    geom_point(size = 4, 
+               color = 'grey',
+               fill = 'black',
+               alpha = 0.8,
+               pch = 21) +
     labs(x = 'Year',
          y = 'Hearings resulting in grant',
          title = 'Version 4',
-         caption = 'Fitted line estimated with a linear regression') +
+         caption = paste0('Fitted line smoothed with a local regression', '\n', 
+                          'with bands representing standard errors')) +
     theme_sfi()
   
-  # version 4
-  g5 <- ggplot(data, 
-               aes(x = year, 
-                   y = number_of_grants)) +
+  
+  
+  # version 5
+  g5 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = number_of_grants)) +
     geom_smooth(method = 'loess',
-                size = 0.5, 
+                fill = 'black',
                 alpha = 0.6,
-                color = 'black',
-                linetype = 2) + 
+                linetype = 0,
+                fullrange = TRUE) + 
     geom_point(size = 3, 
-               alpha = 0.6) +
+               color = 'black',
+               fill = '#B2B2B2',
+               alpha = 0.8,
+               pch = 21) +
     labs(x = 'Year',
          y = 'Hearings resulting in grant',
          title = 'Version 5',
-         caption = 'Fitted line smoothed with a local regression') +
-    theme_sfi()
+         caption = paste0('Fitted line smoothed with a local regression', '\n', 
+                          'with bands representing standard errors')) +
+    theme_sfi(bc = 'white')
   
   
   # version 6
-  g6 <- ggplot(data, 
-               aes(x = year, 
-                   y = number_of_grants)) +
-    geom_bar(stat = 'identity', 
-             alpha = 0.8,
-             fill = 'white',
-             color = 'black') +
-    geom_text(aes(label = number_of_grants),
-              size = 1.8, 
-              color = 'black',
-              nudge_y = 25) +
+  g6 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = number_of_grants)) +
+    geom_smooth(method = 'loess',
+                alpha = 0.6,
+                fill = 'black',
+                linetype = 0) + 
+    geom_point(size = 3, 
+               colour = 'white',
+               alpha = 1,
+               pch = 1) +
     labs(x = 'Year',
          y = 'Hearings resulting in grant',
-         title = 'Version 6') +
-    theme_sfi()
+         title = 'Version 6',
+         caption = 'Fitted line smoothed with a local regression') +
```

Comment:
It looks to me like theyre is not so much as fitted "line" as much as a fitted "range" or "ribbon" (ie, the standard errors). So, perhaps change the language accordingly.

Reply:
(ie, "with bands representing standard errors" as in version 5)